packages/react-magma-dom/src/components/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { XOR } from '../../utils';
+import { css } from '@emotion/core';
+import { transparentize } from 'polished';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { Button, ButtonProps, ButtonSize } from '../Button';
+import { ButtonColor } from '../Button';
+import { IconButton } from '../IconButton';
+import { IconProps } from 'react-magma-icons';
+import { ToggleButtonGroupContext } from '../ToggleButtonGroup/ToggleButtonGroup';
+
+export interface ToggleButtonTextProps extends ButtonProps {
+  /**
+   * Sets a disabled state for a button.
+   */
+  disabled?: boolean;
+  /**
+   * @internal
+   */
+  isChecked?: boolean;
```

Comment:
If someone is using a single ToggleButton that is initialized as check, how do we expect them to do that?

Reply:
I didn't know if that was a requirement, but I added it in.

Reply:
What I meant was, what code do we expect users to use in order to start a single toggle button as checked. Is it something like this, or something else?

```
<ToggleButton isChecked>My Toggle</ToggleButton>
```

Reply:
I made it work with `isChecked`.

Reply:
If that's the case, then it's not an `internal` prop. Then, we should make sure we have an example of this in the docs.